spec/models/workshop_project_click_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'V1::WorkshopProjectClick', type: :request do
+  describe 'GET /v1/projects/:id/redirect/:type_of' do
+    let(:project) { create(:workshop_project) }
+
+    # override in subtests
+    let(:headers) { {} }
+
+    before { get "/v1/projects/#{project.id}/redirect/live", headers: headers }
+
+    it 'redirects to correct url' do
+      expect(response.status).to eq(302)
+      expect(response.headers['Location']).to eq(project.live_url)
+    end
+
+    it 'creates a WorkshopProjectClick' do
+      click = WorkshopProjectClick.last
+
+      # idk
+      expect(click.workshop_project).to eq(project)
+      expect(click.ip_address).to eq('127.0.0.1')
+      expect(click.type_of).to eq('live')
```

Comment:
Yeah, this looks good. No need for the `# idk` comment. Was there something specific here I can help with?